torchaudio/prototype/models/hdemucs/__init__.py

```diff
@@ -0,0 +1,24 @@
+import torchaudio
+
+functions = ["HDemucs", "hdemucs_high", "hdemucs_medium", "hdemucs_low"]
+
+
+def __getattr__(name: str):
+    if name in functions:
+        import warnings
+
+        warnings.warn(
+            f"{__name__}.{name} has been moved to torchaudio.models.hdemucs",
+            DeprecationWarning,
+        )
+
+        from torchaudio import models
+
+        demucs = eval(f"models.{name}")
```

Comment:
`eval` is frowned upon for the security.

Doesn't `return getattr(torchaudio.models, name)` work?